pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -132,29 +156,31 @@ public String toExplainString() {
   @Override
   protected TransferableBlock getNextBlock() {
     if (_hasConstructedAggregateBlock) {
-      return TransferableBlockUtils.getEndOfStreamTransferableBlock();
+      assert _eosBlock != null;
```

Comment:
Instead of an empty EOS block now we send a block with all the stats. This is a pattern that is repeated in all other operators.